packages/mongodb/src/MongoSchemaGenerator.ts

```diff
@@ -74,7 +74,7 @@ export class MongoSchemaGenerator extends AbstractSchemaGenerator<MongoDriver> {
     await Promise.all(promises);
   }
 
-  async ensureIndexes(options: EnsureIndexesOptions = {}): Promise<void> {
+  async ensureIndexes(options: EnsureIndexesOptions = {}, retryLimit = 3): Promise<void> {
```

Comment:
actually, lets not introduce another parameter, we already have the options there, so lets just add the `retryLimit` inside that
```suggestion
  async ensureIndexes(options: EnsureIndexesOptions = {}): Promise<void> {
    options.retryLimit ??= 3
```

Reply:
Updated!